### DISCUSSION TITLE
Support observability in dubbo 3.1.11

### DISCUSSION THREAD

**Discussion by leohao6762** [apache/dubbo#13881] [Question] [answered]
- [x] I have searched the [issues](https://github.com/apache/dubbo/issues) of this repository and believe that this is not a duplicate.

## 问题如下
- 我是目前使用dubbo3.1.11版本，由于南北向接口使用了resteasy，并做了一些定制化的开发，暂时无法升级到3.2.X；目前想做一些可观测性的指标集成到prometheus，这块有现成的东西吗；翻了一下官方文档也找不到资料

**Comment by songxiaosheng** [comment, marked as answer]
Currently not supported for lower versions. If upgrade is not possible, it is recommended to extend customization for monitoring buried points

**Comment by leohao6762** [comment]
我对dubbo源码不太熟悉，能不能给点实现思路

**Comment by songxiaosheng** [comment]
> 我对dubbo源码不太熟悉，能不能给点实现思路

What is your monitoring system? It can collect some key data for monitoring based on your monitoring,

**Comment by leohao6762** [comment]
Prometheus+Grafana;
